newrelic/core/config.py

```diff
@@ -902,6 +902,12 @@ def default_otlp_host(host):
 _settings.application_logging.forwarding.context_data.enabled = _environ_as_bool(
     "NEW_RELIC_APPLICATION_LOGGING_FORWARDING_CONTEXT_DATA_ENABLED", default=False
 )
+_settings.application_logging.forwarding.context_data.include = _environ_as_bool(
```

Comment:
This `_environ_as_bool` doesn't seem right to me. These aren't booleans these are lists. Perhaps `_environ_as_set`?

Reply:
Fixed